imports/startup/client/routes.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { IndexRoute, Router, Route, browserHistory } from 'react-router'
+
+// route components
+import AboutPage from '/imports/ui/pages/AboutPage'
+import AppContainer from '/imports/ui/containers/AppContainer'
+import HomePage from '/imports/ui/pages/HomePage'
+import LoginPage from '/imports/ui/pages/LoginPage'
+import NotFoundPage from '/imports/ui/pages/NotFoundPage'
+import OrganizationPage from '/imports/ui/pages/organizations/OrganizationPage'
+import OrganizationsPage from '/imports/ui/pages/organizations/OrganizationsPage'
+import PositionPage from '/imports/ui/pages/organizations/positions/PositionPage'
+import SearchPage from '/imports/ui/pages/SearchPage'
+import SettingsPage from '/imports/ui/pages/SettingsPage'
+import UserPage from '/imports/ui/pages/users/UserPage'
+import UsersPage from '/imports/ui/pages/users/UsersPage'
+
+export const renderRoutes = () => (
```

Comment:
Because what we are returning here is itself a react component, just naming it `Router` or something is clearer than naming it something that implies its a function (even though technically react components can be functions).